### ISSUE TITLE
0.bootstrap - Manually run of terraform-validator on a windows platform

### ISSUE THREAD

**Issue by LuizSDCit** [terraform-google-modules/terraform-example-foundation#659]
### TL;DR

I'm using a windows platform and I'm  trying to do the step 5.III that is an instruction in the guide of [Deploying with Cloud Build](https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/0-bootstrap#deploying-with-cloud-build) in the 0.bootstrap.

But the command bellow do not work on windows:

```console
terraform-validator validate bootstrap.json --policy-path="../policy-library" --project <A-VALID-PROJECT-ID>
```
The issue may happened because when the bootstrap.json file is generate on windows by the command:

```console
terraform show -json bootstrap.tfplan > bootstrap.json
```

The file formation is UTF-16 instead of UTC-8
When I run the same command on Linux, the generated file is in UTC-8.

Adding this issue to in a future version we may improve the usage to support windows too.
